securedrop_client/logic.py

```diff
@@ -691,33 +624,32 @@ def send_reply(self, source_uuid: str, msg_uuid: str, message: str) -> None:
             if self.api:
                 self.call_api(
                     self.api.reply_source,
-                    self._on_reply_complete,
-                    self._on_reply_timeout,
+                    self.on_reply_success,
+                    self.on_reply_failure,
                     sdk_source,
                     encrypted_reply,
                     msg_uuid,
                     current_object=(source_uuid, msg_uuid),
                 )
-            else:
-                logger.error('not logged in - not implemented!')  # pragma: no cover
-                self.reply_failed.emit(msg_uuid)  # pragma: no cover
+            else:  # pragma: no cover
+                logger.error('not logged in - not implemented!')
```

Comment:
[nit] "not logged in - not implemented!" -- this message says to me that a developer needs to implement this in offline mode, but I don't think this is true, or useful as an error log. Maybe you could elaborate?

Reply:
This was already there, so I didn't change it.